commands/System/disable.js

```diff
@@ -1,26 +1,10 @@
 exports.run = async (client, msg, [type, name]) => {
-  switch (type) {
-    case "inhibitor": {
-      const inhibitor = client.commandInhibitors.get(name);
-      if (!inhibitor) return msg.sendCode("diff", `- I cannot find the inhibitor: ${name}`);
-      inhibitor.conf.enabled = false;
-      return msg.sendCode("diff", `+ Successfully disabled inhibitor: ${name}`);
-    }
-    case "monitor": {
-      const monitor = client.messageMonitors.get(name);
-      if (!monitor) return msg.sendCode("diff", `- I cannot find the monitor: ${name}`);
-      monitor.conf.enabled = false;
-      return msg.sendCode("diff", `+ Successfully disabled monitor: ${name}`);
-    }
-    case "command": {
-      const command = client.commands.get(name) || client.commands.get(client.aliases.has(name));
-      if (!command) return msg.sendCode("diff", `- I cannot find the command: ${name}`);
-      command.conf.enabled = false;
-      return msg.sendCode("diff", `+ Successfully disabled command: ${name}`);
-    }
-    default:
-      return msg.sendMessage("This will never happen");
-  }
+  const t = { command: "commands", inhibitor: "commandInhibitors", monitor: "messageMonitors", finalizer: "commandFinalizers" }[type];
```

Comment:
Use a longer identifier here, and hoist it because it doesn't use client/msg/ect and it can skip object creation time.